src/test/java/edu/ksu/canvas/attendance/repository/AttendanceRepositoryImplITest.java

```diff
@@ -103,6 +122,15 @@ public void saveInBatches_updateExistingAttendance() {
         
         assertEquals(expectedAttendance, actualAttendance);
     }
+
+    @Test
+    public void getAttendanceCommentsBySectionId_HappyPath() {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Rename this method name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.ome.ksu.edu/coding_rules#rule_key=squid%3AS00100)